roles/custom/matrix-client-element/tasks/setup_install.yml

```
@@ -69,6 +69,14 @@
     owner: "{{ matrix_user_username }}"
     group: "{{ matrix_user_groupname }}"
 
+- name: Ensure Element map style installed
+  ansible.builtin.copy:
```

```suggestion
- name: Ensure Element map style installed
  when: matrix_client_element_location_sharing_enabled | bool
  ansible.builtin.copy:
```

Maybe we need an uninstall task as well

---

If you set `matrix_client_element_location_sharing_enabled` to `false`, style file is not created on the next install. It's the same as with `config.json` file.